Xero.NetStandard.OAuth2/Api/FinanceApi.cs

```diff
@@ -1458,7 +1458,7 @@ public async System.Threading.Tasks.Task<IncomeByContactResponse> GetFinancialSt
 
             if (includeManualJournals != null)
             {
-              requestOptions.QueryParameters.Add("includeManualJournals", includeManualJournals.ToString());
+              requestOptions.QueryParameters.Add("includeManualJournals", includeManualJournals.ToString().ToLower());
```

Comment:
why are we setting it to lower() ?

Reply:
when we convert a boolean value to string, it is taking the value as **T**rue or **F**alse.
PRISM is not able to understand the values if casing is wrong, for that reason we need to convert the values to lower